### ISSUE TITLE
Params.py issue

### ISSUE THREAD

**Issue by XSXIEYCH** [Tomiinek/Multilingual_Text_to_Speech#75]
Hello 
I found the code miss some parameter when I run the project, 
File name: Params.py issue
    mel_normalize_mean
    mel_normalize_variance
But I don't know how I can assign values to them
in addition, I can not run here,(file: tacotron2.py, line:145)
The system prompts that the parameter encoded must be of type Int or long
` def _add_conditional_embedding(self, encoded, layer, condition):
        """Compute speaker (lang.) embedding and concat it to the encoder output."""
        embedded = layer(encoded if condition is None else condition)
        return torch.cat((encoded, embedded), dim=-1) `
so I do some change 
`encoded=encoded.long()`
But I got other error  "index out of range is self"
